src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/AppointmentServiceTest.kt

```diff
@@ -269,4 +269,57 @@ class AppointmentServiceTest {
     }
     assertThat(error.message).contains("Appointment not found [id=$appointmentId]")
   }
+
+  @Test
+  fun `appointment feedback can be submitted`() {
```

Comment:
hmm, I was trying to rephrase this to "submitting appointment feedback results in ?" but failed -- should there be a feedback text as well?